hash/tvochken.xml

```diff
@@ -0,0 +1,170 @@
+<?xml version="1.0"?>
+<!DOCTYPE softwarelist SYSTEM "softwarelist.dtd">
+<!--
+license:CC0-1.0
+-->
+<softwarelist name="tvochken" description="TV Ocha-Ken ROM and barcode cards">
+
+	<software name="tvochken" supported="yes">
+		<description>TV to Ocha-Satsu Ocha-Ken Hotsu to Seikatsu</description>
```

Comment:
Not very confident on this description, would appreciate if someone else could confirm it.

Reply:
テレビとお茶札 お茶犬『ほっ』と生活

Not sure how to properly romanize weird cases like this but it's "ほっ" not "ほつ" so at the very least "Hotsu" is not right. It'd probably be just "Ho" here.

Reply:
On the box art the 札 has カード above it as furigana, so at least that character should probably be romanized as "card".
![91x02IfOR+L _AC_SL1500_~2](https://github.com/mamedev/mame/assets/10584181/d85be304-1ebe-4d9b-9197-470809ac736a)

Reply:
according to https://goiryoku.com/onomatopoeia/holtu/ the 『ほっ』 seems to be the Onomatopoeia for a sigh of relief.
In this music video https://www.uta-net.com/movie/225031/ it is also pronounced as Ho.
The 犬 kanji can be read as Inu or Ken and I think in this case it is Inu, because it's a tea dog (look at the packaging).
My suggestion is "TV to Ocha-Card: Ocha-Inu 'Ho' to Seikatsu" for the romanization and "TV and tea card: Life with tea dog 'Ho'"
The machine translation agrees if you put it in like this "テレビとお茶カード: お茶犬『ほっ』と生活"

Reply:
Ocha-ken is an established character name and anything else would be wrong.

Reply:
whoops
https://en.wikipedia.org/wiki/Ocha-Ken

Reply:
Thanks everyone, will use the last suggestion.